platform/ios/src/UIImage+MGLAdditions.h

```diff
@@ -0,0 +1,9 @@
+#import <UIKit/UIKit.h>
+
+#include <mbgl/sprite/sprite_image.hpp>
+
+@interface UIImage (MGLAdditions)
+
++ (std::unique_ptr<mbgl::SpriteImage>)mbgl_spriteImageFromImage:(UIImage *)image;
```

Comment:
It would be more natural to have an instance method in this case: `-mgl_spriteImage`. (Also, for consistency’s sake, the class prefix is `mgl` rather than `mbgl`.)